### DISCUSSION TITLE
Step by Step Tutorial

### DISCUSSION THREAD

**Discussion by andreonifv** [Nico640/docker-unms#85] [General]
Sorry about that, I'm newer here and I'm trying to install the UISP app on an ubuntu server Rasbpery PI 4 running, I've tried everything but it didn't work

DOCKER is running, but I can't access it via http, by any chance you might have a step by step tutorial on how to install it? 

Please.

Thank you very much in advance

**Comment by Nico640** [comment]
Hi,

what command did you use to start UISP? Normally, after installing docker (https://docs.docker.com/engine/install/ubuntu/), you can simply run
```
docker run \
  -p 80:80 \
  -p 443:443 \
  -p 2055:2055/udp \
  -e TZ=<timezone> \
  -v </path/to/config>:/config \
  nico640/docker-unms:latest
```

where you replace `<timezone>` with the corresponding TZ database name (https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) and `</path/to/config>` with the path where you want to store the UISP data.

e.g.
```
docker run \
  -p 80:80 \
  -p 443:443 \
  -p 2055:2055/udp \
  -e TZ=Australia/Sydney \
  -v /home/ubuntu/uisp:/config \
  nico640/docker-unms:latest
```

UISP should then start and be available under the IP address of your Raspberry Pi and the port 443.
